prestoadmin/main.py

```diff
@@ -348,6 +348,16 @@ def parser_for_options():
         help="default to serial execution method"
     )
 
+    advanced_options.add_option(
+        '--rpm-source',
+        action='store',
+        type='string',
```

Comment:
Add `choices=['facebook', 'teradata'] to reduce the error checking you need to do.